### PR TITLE
Feature/setup style guide

### PR DESCRIPTION
**Pull request by kesiah**
- Creating new variables following redesign style guide (https://ncchat.slack.com/archives/C04G7PQ5QG3/p1689926806604239)
- Changed base unit for the system to 16 pixels instead of 10px, because in Volvo group they are using 16px (https://www.volvogroup.com/experiencesystem/en/#/brand/615bfcc428c818001105b4e3/61a0de32a7047974164f01dd/622b4df837a675408d76ebfd) this can cause, that old styles might look bigger.
- We are keeping some `OLD styles` in file `styles/styles.css` in line 138. We will need to get rid of them as soon as we are working in the blocks. Because those variables are still used in specific blocks.
- We are reading the colour variables in JS files.
- All colour variables start with `--c-mycolorname`, font start with `--f-myfontname` so we know what is that variable containing.

Test URLs:
- Before: https://main--vg-volvotrucks-us--hlxsites.hlx.page/
- After: https://<branch>--vg-volvotrucks-us--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/dealer-locator/dealer-locator.css [hunk 1]
```diff
@@ -982,7 +982,7 @@ main .section.dealer-locator-container > div {
     background: #FFF;
     border-radius: 4px;
     float: right;
```
Comment (jonatan-lledo-netcentric) [inline blocks/dealer-locator/dealer-locator.css:984]: if float is not needed, we can remove it. WDYT?

File: styles/styles.css [hunk 2]
```diff
@@ -56,32 +43,106 @@
...
-  --heading-font-family: var(--ff-volvo-novum-medium);
-  --fixed-font-family: 'Roboto Mono', menlo, consolas, 'Liberation Mono', monospace;
 
+  --font-family-body: var(--ff-volvo-novum);
+  --font-family-heading: var(--ff-volvo-novum-medium);
+  --font-family-fixed: 'Roboto Mono', menlo, consolas, 'Liberation Mono', monospace;
+
+  /* REDESIGN */
+  /* Volvo Broad Pro Digital, 60px,Uppercase, 3.6px,#53565A */
+  --f-large-title--font-size: 3.75rem;
```
Comment (jonatan-lledo-netcentric) [inline styles/styles.css:58]: please fix lintern issue first

**Review by jonatan-lledo-netcentric — CHANGES_REQUESTED** [review]
fix lintern issues. aside from this, it's ok to me

**Review by manuel-vara — CHANGES_REQUESTED** [review]
Please do not push your updates to main branch in the parent repo yet. Push to the forked one: https://github.com/Netcentric/vg-volvotrucks-us-rd

**Pull request closed without merging**